File: 品味Java的21種設計模式/dp_java/bridge/example2/CommonMessageSMS.java
package cn.javass.dp.bridge.example2;
/**
 * 以站内短消息的方式发送普通消息
 */
public  class CommonMessageSMS implements Message{

	public void send(String message, String toUser) {
		System.out.println("使用站内短消息的方式，发送消息'"+message+"'给"+toUser);
	}
}

File: 品味Java的21種設計模式/dp_cpp/bridge/example4/CommonMessageSMS.cpp
#include "CommonMessageSMS.h"

namespace cn
{
	namespace javass
	{
		namespace dp
		{
			namespace bridge
			{
				namespace example4
				{

					void CommonMessageSMS::send(std::string message, std::string toUser)
					{
						puts("使用站内短消息的方式，发送消息'"+message+"'给"+toUser);
					}
				}
			}
		}
	}
}